doc/platform-optimization/README.md

```diff
@@ -0,0 +1,188 @@
+1. Add the NRI plugins repository
+   ```bash
+   helm repo add nri-plugins https://containers.github.io/nri-plugins
+   ```
+
+2. Install the balloons resource policy and patch container runtime's
+   configuration on host in order to switch NRI support on.
+   ```bash
+   helm install balloons nri-plugins/nri-resource-policy-balloons --set patchRuntimeConfig=true
+   ```
+
+Now the balloons policy is managing node resources in the cluster as a
+DaemonSet that communicates with the container runtime on every
+node. You should see `nri-resource-policy-balloons-...` pod running on
+every node.
+
+## Configure
+
+Edit the default balloons policy:
+```bash
+kubectl edit balloonspolicy default
+```
+
+Let us consider isolating AI inference and reranking containers in
+[ChatQnA](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA)
+application's Gaudi accelerated pipeline.
+
+In the
+[manifest](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/kubernetes/manifests/gaudi/chatqna.yaml)
+there are "tgi" and "tei" containers that will need a lot of CPUs.
+
+We will allocate 16 CPUs (8 full CPU cores with two hyperthreads on
+each core) for each tgi container, and 32 CPUs (that is 16
+hyperthreaded cores) for each tei container. This is done with the
+following balloons policy configuration that is explained below.
+
+```yaml
+apiVersion: config.nri/v1alpha1
+kind: BalloonsPolicy
+metadata:
+  name: default
+spec:
+  allocatorTopologyBalancing: true
+  balloonTypes:
+  - name: tgi
+    allocatorPriority: high
+    minCPUs: 16
+    minBalloons: 1
+    preferNewBalloons: true
+    hideHyperthreads: true
+    matchExpressions:
+    - key: name
+      operator: Equals
+      values: ["tgi"]
+  - name: tei
+    allocatorPriority: high
+    minCPUs: 32
+    minBalloons: 1
+    preferNewBalloons: true
+    hideHyperthreads: true
+    matchExpressions:
+    - key: name
+      operator: Equals
+      values: ["tei"]
+  - allocatorPriority: normal
+    minCPUs: 1
+    hideHyperthreads: false
+    name: default
+    namespaces:
+    - "*"
+    shareIdleCPUsInSame: numa
+  instrumentation:
+    httpEndpoint: :8891
+    prometheusExport: true
+    reportPeriod: 60s
+    samplingRatePerMillion: 0
+  log:
+    source: true
+    debug: ["policy"]
+  pinCPU: true
+  pinMemory: false
+  reservedPoolNamespaces:
+  - kube-system
+  reservedResources:
+    cpu: "2"
+```
+
+The balloons policy creates "balloons" of CPUs that only containers
+assigned into a balloon are allowed to use.
+
+The most important options in the above configuration example are:
+
+- `allocatorTopologyBalancing: true`. This option ensures that
+  balloons (sets of allowed CPUs) are balanced between CPU sockets in
+  the system, and local NUMA nodes if the system is running a sub-NUMA
+  clustering (SNC) mode. Without this options balloons would be
+  tightly packed on a single socket allowing the other CPU socket
+  sleep and save power. Here we have optimized for performance.
```

Comment:
s/sleep/to sleep.

Add a sentence, "To optimize for power savings, one could alternately have set allocatorTopologyBalancing: false"

Reply:
Thank you, added!